gradle.properties

```diff
@@ -80,6 +80,8 @@ oel.publication=true
 # Look for `COMPOSE` in libraryversions.toml
 oel.androidx.version=1.3.3
 # Look for `COMPOSE_MATERIAL3` in libraryversions.toml
-oel.androidx.material3.version=1.0.2
+oel.androidx.material3.version=1.0.1
```

Comment:
1.0.1 is the latest published stable version - https://developer.android.com/jetpack/androidx/releases/compose-material3